src/Tests/Microsoft.NET.Publish.Tests/GivenThatAPublishedDepsJsonShouldContainVersionInformation.cs

```diff
@@ -149,7 +149,7 @@ void testProjectChanges(TestProject testProject)
             immutableDir.Should().BeEquivalentTo(publishDir, "published immutable collections library from should win");
         }
 
-        [Fact]
+        [Fact(Skip = "https://github.com/dotnet/core-setup/issues/4146")]
```

Comment:
@dsplaisted Since --fx-version will for the use the exact version. RuntimeFrameworkVersion property is 2.1 but what on the disk is 2.1.1. The similar reason why the test above it is disabled.

Reply:
I think instead of disabling these tests, you can change GetRollForwardNetCoreAppVersion to set the TargetLatestRuntimePatch property to true in the project it creates.  Then it will return 2.1.1 which is what you want.

Reply:
That works 👍